_includes/sections/working-here/your-pay-pension-and-other-benefits.md

```diff
@@ -108,38 +108,100 @@ your holiday allowance shown on your BreatheHR dashboard.
-current role we will consider if there is a suitable alternative role. If we are
-unable to reach an agreement that will permit you to return to work, we will
-give consideration to your possible dismissal.
-
-We will always only consider dismissal after we have exhausted all other
-reasonable options.
+This policy covers both short and longer term sickness absences. Our aim is to
+look after the health and wellbeing of all dxw staff and the needs of the
+company.
+
+##### Communication of sickness
+
+If you’re sick, you must let your line manager or a member of the HR team know
+by 10:00am or as soon as you reasonably can after that. This can be done via
+email, directly on Slack, phone call or text. Line managers will record and
+monitor sickness for individuals within their teams.
+
+If you are unwell and know that you are going to be off for a period of time, or
+have been signed off by your GP until a certain date, you don’t need to update
+your line manager daily. Let them know when you will be checking in again so
+they know when to expect to hear from you.
+
+When you return to work, you must check your sickness has been correctly
+recorded in BreatheHR, update it if necessary and mark yourself as returned to
+work. Your line manager will then review and close the sick leave. If you're
+sick for more than 7 calendar days in a row, you'll also need to provide a fit
+note.
+
+##### Absence meetings
+
+Your line manager should check-in with you on a regular basis about your
+wellbeing, and catch up with you after any sickness absence to see how you are.
+If you're sick for more than 7 working days over a rolling 12 month period, your
+line manager will invite you to an absence meeting to discuss your health. Where
+appropriate, we’ll talk about how we can support your return to work and any
+temporary or permanent adjustments which might help improve your wellbeing and
+reduce your levels of absence. We might also agree an action plan and / or a
+review period. Where we think it would be helpful, we’ll seek medical advice via
+an Occupational Health advisor.
+
+If you have a long term health condition and would like to discuss reasonable
+adjustments and ways of working that will help, we welcome a conversation at any
+point. The sooner we know, the sooner we can support you.
+
+##### Pay
+
+Within the 12 month period, a person will receive:
+
+- full pay for the first 12 days of sickness absence
+- half pay (but no less than living wage) for the next 8 days of sickness
+  absence
+- living wage for the next 10 days of sickness absence
+- statutory sick pay pay after 30 days of sickness absence
+
+There'll be circumstances where we'll make exceptions, because it's not possible
+to write a policy that is fair in every situation. This structure will form our
+starting point for those decisions. All decisions on exceptions will be made by
+the Directors’ group. Our aim is to ensure that we’re treating everyone fairly
```

Comment:
```suggestion
the Directors’ group after discussion with you and your line manager. Our aim is to ensure that we’re treating everyone fairly
```

It could do with some detail on how me, the employee, can request an exception. Do I talk to my line manager? What do they do? Do they talk to the directors on my behalf? Do they talk to HR? Does someone schedule a meeting including me, or not? etc.

Reply:
Every situation may be different and setting guidelines here would be unrealistic, ie you may not be able to attend the meeting. A line manager would be a aware of a situation (prolonged sickness) then a conversation about this would have happened/be happening.

Reply:
Every circumstance is different so it may not depend on the outcome of the absence meeting. We can't include a list of things that may require an extension.

Reply:
Will the directors ever make a decision without consulting you or your line manager? Is there an issue with my suggestion?

```suggestion
the Directors’ group after discussion with you and your line manager. Our aim is to ensure that we’re treating everyone fairly
```